paper.tex

```diff
@@ -54,49 +76,99 @@ \section*{Introduction}
-implement, read, and share. Models are then represented in an unambiguous
-low-level format within Python that checks model validity. The initial Python
-release of \demes\ (version 0.1, described here) supports \emph{static}
-demographic descriptions, that is, models with fixed parameters. Future
-releases will support [[drawing parameters from distributions, for example for
-ABC-based inference, or describing ``dynamic`` models where parameters may be
-fit to data, e.g. \moments\ or \dadi]].
-
-\section*{The \demes\ specification}
+This ability to infer and simulate such complex demographic scenarios, however,
+has highlighted a major shortcoming in community standards. The fragmented
+landscape of different approaches to describing demographic models makes
+it difficult to compare inferences made by different methods,
+and to reliably simulate from previously inferred models.
+
+Inference methods for demography usually output a set of
+text files which describe the inferred model,
+and nearly all methods use different file formats.
+% Thus, any comparison
+% between inferences requires these file formats to be parsed, with the user
+% being left with the task of unifying the data models.
+Inference results are typically reported in publications
+via a combination of visual depiction,
+a list of key parameters in tabular form and a discussion within the text.
+Unfortunately there is often ambiguity in these descriptions and
+implementing the precise model inferred for later simulation
+is at best tedious and error prone~\citep{ragsdale2020lessons}
+and occasionally impossible because of missing information.
+The \stdpopsim\ project is a community-maintained catalog of species
+information and demographic models designed to provide standardised
+population genetic simulations~\citep{adrion2020community}.
+The development process for adding demographic models to
+\stdpopsim\ tries to avoid errors in model implementation by requiring two
+independent implementations of each model
+to be equivalent. While this process is
+robust and has uncovered both significant
+errors~\citep{ragsdale2020lessons} and ambiguities in
+published models, it is labour intensive.
+It would be far better if published results
+were reported in an unambiguous standardised format that could be provided
+directly as input to simulators.
+
+Simulation is a core tool in population genetics, and a
+large number of different methods have been developed
+over the past three decades~\citep{carvajal2008simulation,liu2008survey,
+arenas2012simulation,yuan2012overview,hoban2012computer}.
+Simulations are based on highly idealised population models
+and one of the key uses of inferred demographic histories
+is to make simulations more realistic.
+Simulation methods take three broad approches to specifying
+the demographic model to simulate,
+using either a command line
+% JK: is there a simulator that just uses a CLI that's not using ms format?
```

Comment:
Maybe [discoal](https://github.com/kr-colab/discoal)? `-ed` and `-ea` for population join and split don't look like ms commands to me.